.github/workflows/sdk-publish.yaml

```diff
@@ -16,6 +16,10 @@ on:
         required: false
         type: string
         default: "5.x"
+      target:
+        description: "Generate a specific target by name"
```

Comment:
Not sure the description even appears anywhere, but still, this doesn't seem right